src/scicookie/{{cookiecutter.project_slug}}/{{cookiecutter.package_slug}}/__init__.py

```diff
@@ -22,6 +25,7 @@ def get_version():
 
 version = get_version()
 
-__author__ = {{ QUOTE }}{{ cookiecutter.author_full_name }}{{ QUOTE }}
-__email__ = {{ QUOTE }}{{ cookiecutter.author_email }}{{ QUOTE }}
 __version__ = version
+{%- endif %}
+__author__ = {{ QUOTE }}{{ cookiecutter.project_version }}{{ QUOTE }}
```

Comment:
@Saransh-cpp Why did you change this?

Reply:
ohhh good catch!! and sorry for my dynamic reading in the review XD
I really need some vacations! I can fix that!

Reply:
Woops, I completely missed that. Thank you for fixing the mess, @xmnlab and @Anavelyz!